common/hash.go

```diff
@@ -18,8 +18,19 @@ package common
 
 import (
 	"golang.org/x/crypto/blake2b"
+	"golang.org/x/crypto/sha3"
 )
 
+// Returns the 128-bit blake2b hash of the input data
```

Comment:
Comment needs to start with name of function `Blake2b128`

Reply:
done